users/vosechu/vosechu.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "quantum.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+enum userspace_custom_layers {
+  DV = 0,
+  QW,
+  GAM1,
+  RSE,
+  LWR,
+  LFT,
+  MOUSE
+};
+
+enum userspace_custom_keycodes {
+  PAWFIVE = SAFE_RANGE,
+  MOUKEY,
+  MS_BTN1,
+  MS_BTN2,
+  MS_BTN3
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define _LAYER_ KC_TRNS
+#define XXXXXXX KC_NO
+
+// Adding macros to make the keymaps below much easier to read.
+#define CTLESC CTL_T(KC_ESC)
```

Comment:
I would recommend using an underscore in most of these, just to make them more readable.  But that's entirely optional.

Reply:
Well, this is optional.   If you're fine with it, then feel free to leave it.

But as somebody that does spend a LOT of time in QMK....  I find that underscores do make it a bit more readable.  

And I don't know about you, but I've come back to my own code and wondered WTF I was doing. :D 
So making sure it's readable is a good thing. But it's more important that it's readable TO YOU.